### ISSUE TITLE
DCD Reader timeseries does not read backwards

### ISSUE THREAD

**Issue by jdetle** [MDAnalysis/mdanalysis#922]
### Expected behaviour

In creation of a DCD timeseries, a step of -1 should yield a timeseries that travels backwards across a DCD file. 
### Actual behaviour

The DCD timeseries will behave as if a step of 1 was given, and the increment from the start.
### Code to reproduce the behaviour

``` python
import MDAnalysis as mda
u = mda.Universe(PSF, DCD)
atoms = u.select_atoms('backbone and name CA')
ts =  u.trajectory.timeseries(atoms)
ts_negative = u.trajectory.timeseries(atoms, start=5, stop=1, step=-1)

# (ts[:, 5:1:-1, :] != ts_negative
# ts_negative == ts[:, 5:9:1, :] 
```
### Current reason for this behavior

Trying to grok the C code. Give me a few ~~hours~~, ~~days~~, months 
### Current version of MDAnalysis:

0.15.1-dev0

**Comment by orbeckst** [comment]
This was fixed with #1372:
```python 
>>> np.allclose(ts[:, 5:1:-1, :], ts_negative)
True
```
(and this test was run under Python 3 ;-))

**Comment by orbeckst** [comment]
@kain88-de I am awarding this one to you post-closure ;-)
